main.nf

```diff
@@ -316,7 +323,7 @@ if(params.notrim){
 
         input:
         set val(name), file(reads) from read_files_trimming
-        file wherearemyfiles
+        file wherearemyfiles from wherearemyfiles_1
```

Comment:
Same here, please use explicit names for channels :-) 
It gets super confusing at some point otherwise....